backend/entityservice/tests/test_deletes.py

```diff
@@ -0,0 +1,45 @@
+from entityservice.tests.util import post_run, delete_project, create_project_upload_fake_data, delete_run, \
+    get_project_description, get_run, wait_for_run_completion
+import pytest
+
+
+def test_delete_project_after_creating_run_with_clks(requests, result_type_number_parties):
+    result_type, number_parties = result_type_number_parties
+    project, _ = create_project_upload_fake_data(
+        requests, [100] * number_parties, overlap=0.5, result_type=result_type)
```

Comment:
Instead of the three first lies, could use `_create_data_linkage_run(requests, result_type_number_parties)`